xdn-spartacus-example/README.md

```diff
@@ -0,0 +1,27 @@
+# XdnSpartacusExample
```

Comment:
Is all of this information still applicable? If so, we can keep it, but let's also update it to highlight the XDN. The other examples have a pretty simple readme you can use as a guide: https://github.com/moovweb-docs/xdn-examples/blob/main/xdn-next-example/README.md